hydromt/data_catalog/data_catalog.py

```diff
@@ -1006,10 +1006,11 @@ def export_data(
         self,
         data_root: Union[Path, str],
         bbox: Optional[Bbox] = None,
-        time_tuple: Optional[TimeRange] = None,
+        time_range: Optional[TimeRange] = None,
```

Comment:
Seems like you partially updated the `export_data` function to v1, but not fully?

Reply:
Yeah, that's true, these are some left over changes from before I decided to leave the export_data function alone, I can remove them if you like

Reply:
no need